test/suites/parathreads/test_tanssi_parathreads.ts

```diff
@@ -259,23 +386,26 @@ describeSuite({
                 expect(Number(await context.web3().eth.getBalance(randomAccount.address))).to.be.greaterThan(0);
             },
         });
+        /* Commented out as block generation frequency for parathread is
```

Comment:
From my tests it seems the block frequency is about 2x slower than expected because of slot time, so maybe we can add this test back but with a high margin of error.